scripts/CEE/worker-node-operations/script.sh

```diff
@@ -106,11 +106,11 @@ drain_worker(){
         fi
     fi
 
-    if [ $? -eq 0 ]; then 
-        echo "[OK] Node ${WORKER} drained successfully."
-        echo
+    if oc adm cordon "${WORKER}"; then
+        echo "[OK] Node ${WORKER} cordoned successfully"
```

Comment:
Why changed this from drained to cordoned?

Reply:
I changed it according to the command. Reverted the change